FreeRTOS/Lab-Project-FreeRTOS-FAT
# Fix build combination and spelling CI checks

Description
-----------
Fix build combination and spelling CI checks

Test Steps
-----------
N/A

Checklist:
----------


- [ N/A ] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
